tests/unit/aiplatform/test_datasets.py

```diff
@@ -1045,7 +1098,16 @@ def test_tabular_dataset_column_name_bq_with_creds(self, bq_client_mock):
     def test_tabular_dataset_column_name_bigquery(self):
         my_dataset = datasets.TabularDataset(dataset_name=_TEST_NAME)
 
-        assert my_dataset.column_names == ["column_1", "column_2"]
+        assert my_dataset.column_names == set(
```

Comment:
@sasha-gitg Using a set means that when I write a unit-test, I don't have to know about the implementation details on how the list is ordered.

There are workarounds, but using a set seems cleanest.